src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -633,7 +634,7 @@ public void run() {
          * Occupies {@link Executor} while workflow uses this slave.
          */
         @ExportedBean
-        private final class PlaceholderExecutable implements ContinuableExecutable {
+        public final class PlaceholderExecutable implements ContinuableExecutable {
```

Comment:
Why do we have to make the class public?

Reply:
I thought it needed to be public to use `@ExportedBean`, but that seems to be incorrect

Reply:
Thanks for checking! I wasn't sure either way but figured that since it had `@ExportedBean` when it was private then maybe it didn't need to be public. If you are curious about the details, the exporting code appears to be in [Stapler's org.kohsuke.stapler.export package](https://github.com/stapler/stapler/tree/cafcb607e9e957bc8f57782eab3b0568a2ab3f42/core/src/main/java/org/kohsuke/stapler/export), although I have never looked at the code in detail.

Reply:
I'm also not sure. I thought you knew. I've only seen `@ExportedBean` with public classes, but that doesn't mean its correct